src/Timer.ts

```diff
@@ -167,17 +168,35 @@ class Timer<T = void>
     return this._status;
   }
 
+  /**
+   * Timestamp when this is scheduled to finish and execute the handler
+   * Guaranteed to be weakly monotonic within the process lifetime
+   * Compare this with `performance.now()` not `Date.now()`
+   */
+  public get scheduled(): Date | undefined {
+    return this.scheduled_;
+  }
+
+  /**
+   * Delay in milliseconds
+   * This may be `Infinity`
+   */
+  public get delay(): number {
+    return this.delay_;
+  }
```

Comment:
Do note that we actually use `this._x` when there's a `get x()`. See `KeyRing.ts`.

Reply:
Fixed